app/src/main/java/com/example/bluromatic/ui/theme/Theme.kt

```diff
@@ -62,35 +55,9 @@ fun BluromaticTheme(
         darkTheme -> DarkColorScheme
         else -> LightColorScheme
     }
-    val view = LocalView.current
-    if (!view.isInEditMode) {
-        SideEffect {
-            setUpEdgeToEdge(view, darkTheme)
-        }
-    }
```

Comment:
Please revert this to the initial code snippet

Reply:
val view = LocalView.current
    if (!view.isInEditMode) {
        SideEffect {
            val window = (view.context as Activity).window
            window.statusBarColor = colorScheme.primary.toArgb()
            WindowCompat.getInsetsController(window, view).isAppearanceLightStatusBars = darkTheme
        }
    }